src/projects/diff/diff.html

```diff
@@ -113,6 +113,11 @@
       on-confirm="_addGroup"
     ></input-dialog>
 
+    <move-dialog id="moveGroupDialog"
+      action-title="Move commit to another group"
```

Comment:
The word `commit` is not really correct in this context. The user moves a small part of a commit, which is formally called `hunk` (or just `change`).